README.md

```diff
@@ -45,12 +45,13 @@ If this is in the file `/path/to/foo/foo.cpp`, then you can compile and run this
 
 ```bash
 > cd /path/to/foo
-> clang++ -std=c++1y -I /path/to/stan-math -I /path/to/Eigen -I /path/to/boost -I /path/to/sundials -I /path/to/tbb -L /path/to/tbb-libs -ltbb -D_REENTRANT  foo.cpp
-> ./a.out
+> make -f path/to/stan-math/make/standalone math-libs
+> make -f path/to/stan-math/make/standalone foo
+> ./foo
 log normal(1 | 2, 3)=-2.07311
 ```
 
-The `-I` includes provide paths pointing to the five necessary includes:
+The first make command with the `math-libs` target ensures that all binary dependencies are build and ready to use. The second make command ensures that all dependencies of Stan Math are available to the compiler which are:
```

Comment:
"... all binary dependencies are built ..."

Split the second sentence in two: "The second make command ensures that all dependencies of Stan Math are available to the compiler. These are:"